src/Button/ButtonProps.tsx

```diff
@@ -0,0 +1,12 @@
+import { ReactNode, MouseEventHandler } from 'react';
+
+export interface ButtonProps {
```

Comment:
This should be a `.ts` file, since it doesn't contain any jsx

Reply:
Fixed through d075208!